src/components/course/data/utils.jsx

```diff
@@ -682,9 +679,5 @@ export const getCourseStartDate = ({ contentMetadata, courseRun }) => {
     startDate = courseRun?.start;
   }
 
-  if (startDate) {
-    return moment(startDate).format(DATE_FORMAT);
-  }
-
-  return undefined;
+  return startDate;
```

Comment:
note: the consumer of this function is responsible for formatting the date; this function should just return the date string. That way, components can format the date differently as needed.